apps/block_scout_web/assets/css/theme/_trustlines_variables.scss

```diff
@@ -1,7 +1,5 @@
-// general
-$primary: #0f3255;
-$secondary: #f5faff;
-$tertiary: #1e8cc3;
+// Avoid redundancy and re-use the non-critical variables.
```

Comment:
what are non critical variables?

Reply:
This is a `preload` content. I can't find any extended explanation for its introduction. The initial commit message of this is: `Isolating staking dapp css, extracting of non-critical css`.
After all, some components rely on the non-critical definitions and can't/don't use the "normal" stylesheet,. which become loaded later. So do the modals.